PReLU.cu

```diff
@@ -0,0 +1,263 @@
+#include "THCApply.cuh"
+#include "THCReduce.cuh"
+#include "utils.h"
+
+// This is bad, because the following is defined in SpatialConvolution.cu ...
```

Comment:
btw would it be better to separate the compilation of each file in cunn? take some time now